api/models/topics.models.js

```diff
@@ -23,9 +23,37 @@ function selectArticleById(article_id) {
 		});
 }
 
-function selectArticles(){
-  return db.query(`SELECT author, title, article_id, topic, created_at, votes, article_img_url FROM articles ORDER BY created_at DESC;`).then((result) => {
-    return result.rows
-  })
+function selectArticles() {
+	return db
+		.query(
+			`SELECT 
+    articles.author, 
+    articles.title, 
+    articles.article_id, 
+    articles.topic, 
+    articles.created_at, 
+    articles.votes, 
+    articles.article_img_url, 
+    COUNT(comments.article_id) AS comment_count
+FROM 
+    articles
+INNER JOIN 
+    comments 
+ON 
+    articles.article_id = comments.article_id
+GROUP BY 
+    articles.author, 
```

Comment:
this group by clause is unnecessarily long! have another look at how to do this later

Reply:
I thought it was !! haha, have updated now, I had an earlier issue and for some reason that was the only way to resolve, have updated and it all works now